src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -71,6 +71,28 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                         If Not CaseInsensitiveComparison.Equals(valueText, idText) Then
                             Return newNode.ReplaceToken(literal, CreateDecimalLiteralToken(literal, valueText, value))
                         End If
+                    Case SyntaxKind.IntegerLiteralToken
+                        ' Get the literal identifier text which needs to be pretty listed.
+                        Dim idText = literal.GetIdentifierText()
+                        Dim value As Long = CType(literal.Value, Long)
+
+                        If value = 0 Then
+                            ' Overflow/underflow case or zero literal, skip pretty listing.
```

Comment:
@jasonmalinowski in the case of an overflow the value comes through as zero. It does look like we could get at the diagnostics for the token and check for this. Neither decimal nor float values reduce 00.0 literals

@AnthonyDGreen When using ULong I am unable to use the Convert.ToString function to create strings in the Hex or Oct bases. I could use ToString("X") to convert to Hex. However, there isn't a built in framework function to convert to Oct.